test/raygun_unhandled_rejection_test.js

```diff
@@ -0,0 +1,36 @@
+const util = require("util");
+var childProcess = require("child_process");
+
+var test = require("tap").test;
+
+var { makeClientWithMockServer } = require("./utils");
+const { deepEqual } = require("node:assert");
+
+test("reporting uncaught exceptions", async function (t) {
+  const testEnvironment = await makeClientWithMockServer();
+  const messagePromise = testEnvironment.nextRequest();
+
+  await util
```

Comment:
A code comment here to clarify would be good to add

Reply:
I will add it.
FYI this was copied from `test/raygun_uncaught_exception_test.js` which had no comment here either.